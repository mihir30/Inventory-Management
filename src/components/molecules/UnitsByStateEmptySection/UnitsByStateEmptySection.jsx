import React from 'react'
import './UnitsByStateEmptySection.scss'
const UnitsByStateEmptySection = () => {
  return (
      <div className="table-overlay"><p className='empty-table-text'>Nothing here</p></div>
  )
}

export default UnitsByStateEmptySection