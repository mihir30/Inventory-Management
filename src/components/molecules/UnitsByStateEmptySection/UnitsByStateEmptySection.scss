.table-overlay{
    text-align: center;
    font-size: 32px;
    color: #94A1B3;
}

.empty-table-text{
    margin-top: 45px;
  

}