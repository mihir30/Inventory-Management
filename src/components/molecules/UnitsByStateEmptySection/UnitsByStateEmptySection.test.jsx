import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDom from "react-dom";
import UnitsByStateEmptySection from './UnitsByStateEmptySection';

describe("Units by State empty section Component", () => {

    it('renders the component properly', () => {
        const divEl = document.createElement("div");
        ReactDom.render(<BrowserRouter><UnitsByStateEmptySection/></BrowserRouter>, divEl); 
        
    });

    it('should have the right units by state empty section component snapshot', () => {
        const tree = renderer.create().toJSON();
        expect(tree).toMatchSnapshot();
    })

});