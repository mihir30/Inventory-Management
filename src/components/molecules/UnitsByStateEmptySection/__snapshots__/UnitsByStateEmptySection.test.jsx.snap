// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Units by State empty section Component should have the right units by state empty section component snapshot 1`] = `null`;
